colorfield/static/colorfield/colorfield.js

```diff
@@ -1,12 +1,16 @@
-/** global: django */
-
-window.addEventListener('load', function (event) {
-    if (typeof django !== 'undefined' && typeof django.jQuery !== 'undefined') {
-        (function ($) {
-            // add colopicker to inlines added dynamically
-            $(document).on('formset:added', function onFormsetAdded(event, row) {
-                jscolor.install();
-            });
-        })(django.jQuery);
+window.addEventListener('load', function () {
+  const inputs = document.getElementsByClassName('colorfield_field coloris');
+  for (const input of inputs) {
+    try {
+      const id = input.getAttribute('id');
+      // convert python dict into json object
+      const options = input.getAttribute('data-coloris')
+        .replaceAll("'", '"')
+        .replaceAll('True', 'true')
+        .replaceAll("False", 'false');
```

Comment:
This is really bad, just need to parse the JSON previously passed to the data attribute, without touching the data string manually.